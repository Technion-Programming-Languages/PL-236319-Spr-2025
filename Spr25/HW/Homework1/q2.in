neva gonna give u up 12345 never gonna let you down
xqt0 z4xx0 zitq 3 3d c5jfl xqtq7 z4xx0 2qr n43 h49x 
cajf k07pb lrb 70x jc 6i24kqrq78
