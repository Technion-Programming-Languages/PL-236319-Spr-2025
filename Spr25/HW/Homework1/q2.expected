1?34 ?ar?? 5t? ran 31 ?ilo?eter?
From: 12345adegilnoprtuvwy
To:   c5jfl0hqzi2x4d7r3t9n
